cltk/lemmatize/latin/unsupervised.py

```diff
@@ -0,0 +1,382 @@
+            return lemmalist
+        lemmalist = self.lemmatizer.lookup([target])
+        lemmas = self.lemmatizer.isolate(lemmalist)
+        if len(lemmas) > 1:
+            all_lemmas_total = sum([self.count_dictionary[l] for l in lemmas])
+            try:
+                lemmalist = [(l, (self.count_dictionary[l] / all_lemmas_total)) for l in lemmas]
+            except ZeroDivisionError:
+                print([(self.count_dictionary[l], l) for l in lemmas])
+            return lemmalist
+        lemmalist = []
+        lemmaobj = (lemmas[0], 1)
+        lemmalist.append(lemmaobj)
+        return lemmalist
+
+    def token_cleanup(self, rawtoken):
+        '''Standardize tokens by replaceing j with i and v with u, and
+        split into multiple tokens as needed with tokenize() method of word_tokenizer class
+        parameters
+        ----------
+        rawtoken: the token as drawn from the text
+        return
+        ------
+        tokenlist: a list of possible word or punctuation tokens
+        '''
+        rawtoken = self.jv.replace(rawtoken)
+        rawtoken = rawtoken.lower()
+        tokenlist = self.word_tokenizer.tokenize(rawtoken)
+        #sometimes words are split into enclitics and punctuation.
+        return tokenlist
+
+    def save_pickle(self, filename):
+        '''Saves the self.count_dictionary object for later reuse.
+        dependencies
+        ------------
+        os package
+        parameters
+        ----------
+        filename: name for the pickle file'''
+        relativepath = join('~', 'cltk_data', 'latin', 'model', 'latin_models_cltk', 'frequency')
+        path = expanduser(relativepath)
+        pickle_file = join(path, filename)
+        if not os.path.isdir(path):
+            os.makedirs(path)
+        pickle.dump(self.count_dictionary, open(pickle_file, "wb"))
+
+    def load_pickle(self, filename):
+        '''Load the self.count_dictionary object saved by save_pickle.
+        dependencies
+        ------------
+        os package
+        parameters
+        ----------
+        filename: name of the pickle file'''
+        relativepath = join('~', 'cltk_data', 'latin', 'model', 'latin_models_cltk', 'frequency')
+        path = expanduser(relativepath)
+        pickle_file = join(path, filename)
+        pickle_file = open(pickle_file, 'rb')
+        self.count_dictionary = pickle.load(pickle_file)
+
+    def train_model(self):
+        '''open all the tesserae files and call read_files() on each to build freq model'''
+        relativepath = join('~', 'cltk_data',
+                            'latin', 'text',
+                            'latin_text_tesserae_collection', 'la')
+        path = expanduser(relativepath)
+        onlyfiles = [f for f in listdir(path) if isfile(join(path, f)) and 'augustine' not in f and 'ambrose' not in f and 'jerome' not in f and 'tertullian' not in f and 'eugippius' not in f and 'hilary' not in f] # pylint: disable=line-too-long
```

Comment:
In a list comprehension, you can break up the lines to make it more readable. Maybe try that here.